source/shared/cpp/AdaptiveCardsSharedModel/AdaptiveCardsSharedModelUnitTest/AdaptiveCardsSharedModelUnitTest.vcxproj

```diff
@@ -156,6 +156,7 @@
     <ClInclude Include="targetver.h" />
   </ItemGroup>
   <ItemGroup>
+    <ClCompile Include="AdditionalPropertiesTest.cpp" />
```

Comment:
>AdditionalPropertiesTest.cpp [](start = 24, length = 28)

This file is not included in the commit...

Reply:
yup, you are correct. I was going to send you e-mail once I felt ready.

---
In reply to: [171353887](https://github.com/Microsoft/AdaptiveCards/pull/1179#discussion_r171353887) [](ancestors = 171353887)